pandas/tests/frame/test_reductions.py

```diff
@@ -765,7 +765,7 @@ def test_operators_timedelta64(self):
     def test_std_timedelta64_skipna_false(self):
         # GH#37392
         tdi = pd.timedelta_range("1 Day", periods=10)
-        df = DataFrame({"A": tdi, "B": tdi})
+        df = DataFrame({"A": tdi, "B": tdi}, copy=True)
```

Comment:
why would copy matter here at all? isn't tdi supposed to be immutable? we need to *always* copy no?

Reply:
> why would copy matter here at all?

there are a couple of inter-twined issues here.  the copy is necesary because we need df["A"] and df["B"] to not share data.  We could do `df = DataFrame({"A": tdi.values, "B": tdi.values.copy()})`

> isn't tdi supposed to be immutable?

yes, but thats kinda orthogonal (maybe 45°).  e.g. in master

```
tdi = pd.timedelta_range("1 Day", periods=3)
df = pd.DataFrame(tdi)
df.iloc[0, 0] = pd.NaT
>>> tdi
TimedeltaIndex([NaT, '2 days', '3 days'], dtype='timedelta64[ns]', freq='D')
```

> we need to always copy no?

i think we do something like this for dt64tz motivated by a similar immutability concern.  ill see if that can be adapted

Reply:
I think this copy=True can be removed now?